src/test/game/RulesTest.java

```diff
@@ -25,36 +25,43 @@ public void setUp() {
         moves.put(PAPER, paperMove);
         moves.put(SCISSORS, scissorsMove);
     }
-    
+
     @Test
     public void scoresADraw() {
 
         Rules rules = new Rules(moves);
 
-        assertEquals(DRAW.getResult(), rules.scoreGameTwo(ROCK, ROCK));
+        assertEquals(DRAW.getResult(), rules.scoreGame(ROCK, ROCK));
```

Comment:
When you apply the refactoring recommended for the `Result` enum this would eliminate the need for `getResult()`. Same would apply for the rest of the tests here.